contracts/container/config.yml

```diff
@@ -1,7 +1,7 @@
 name: "NeoFS Container"
-safemethods: ["alias", "count", "containersOf", "get", "owner", "list", "eACL", "getContainerSize", "listContainerSizes", "iterateContainerSizes", "iterateAllContainerSizes", "version"]
+safemethods: ["alias", "count", "containersOf", "get", "owner", "list", "nodes", "eACL", "getContainerSize", "listContainerSizes", "iterateContainerSizes", "iterateAllContainerSizes", "version"]
 permissions:
-  - methods: ["update", "addKey", "transferX",
+  - methods: ["update", "addKey", "addNextEpochNodes", "commitContainerListUpdate", "transferX",
```

Comment:
This certainly doesn't need to be changed.

Reply:
what do you mean? i am not sure i know the sequence of contract changes. what should be done first, what can be regenerated automatically, what does `permissions` mean?

Reply:
Just don't change anything here.

Reply:
does not explain the problem. found explanation: https://github.com/nspcc-dev/neo-go/blob/master/docs/compiler.md#permissions
dropped

Reply:
Yes, and the definition we have now sucks a bit ("update" can have a specific contract). But that's not worth fixing now.